avocado/utils/pci.py

```diff
@@ -59,6 +59,22 @@ def get_pci_addresses():
         return addresses
 
 
+def get_pci_hbas():
+    """
+    Gets list of PCI hbas, first pci address in a hba, in the system.
+    Does not return the PCI Bridges/Switches.
+    e.g: for pci address 0000:00:14.0 and 0000:00:14.1, we return 0000:00:14.0
+
+    :return: list of full PCI hbas including domain (0000:00:14.0)
+    """
```

Comment:
@ldoktor lspci is always sorted in my understanding. It is safe to assume so.
Also, AFAIU, the first address is always .0

I have modified the code slightly. Please check now. Hope your concerns are taken care
The functionality I am looking is, `[1.0, 1.1] => [1]`, and `[1.1, 1.3] => [1]`